src/test/gcp/cloudfunctionsv2.spec.ts

```diff
@@ -460,8 +462,96 @@ describe("cloudfunctionsv2", () => {
       };
 
       expect(
-        cloudfunctionsv2.functionFromSpec(complexFunction, CLOUD_FUNCTION_V2_SOURCE)
+        await cloudfunctionsv2.functionFromSpec(complexFunction, CLOUD_FUNCTION_V2_SOURCE)
+      ).to.deep.equal(complexGcfFunction);
+    });
+
+    it("should calculate non-trivial fields with a GCS trigger", async () => {
+      const storageStub = sinon
```

Comment:
*nit: I'd personally setup this stub in `beforeAll` and restore in `afterAll` block even when the some test case does not require `getStorageBucket` to be stubbed. That way, I avoid any possibility of pulling my hair our wondering why my test cases are failnig only to realize I forgot to `stub.restore()` after one test case...

Reply:
Fixed